site/zh-CN/reference/index_selection.md

```diff
@@ -0,0 +1,301 @@
+---
```

Comment:
I think we have the CN version of this page?

Reply:
We do, but it needs some revision. I'll update it tomorrow.